Makefile

```diff
@@ -69,3 +69,6 @@ build-sol:
 
 check-resources:
 	poetry run python scripts/check_resources.py
+
+get-blockhashes:
```

Comment:
does this need to run before every test? if so, could we have it ran programmatically?

Reply:
Thanks for the excalidraw diagram Elias! So for testing, we don't need to run it every time, it takes around a minute or two for this script to finish, mainly due to retries from `Too Many Requests`. I thought it would add more time to the testing and decided not to run it programmatically

Reply:
agreed, testing should use a mock file